src/main/java/uk/gov/companieshouse/api/accounts/Kind.java

```diff
@@ -14,7 +14,8 @@
     INTANGIBLE_ASSETS_NOTE("small-full-accounts#intangible-assets-note"),
     SMALL_FULL_STATEMENT("small-full-accounts#statements"),
     STOCKS_NOTE("small-full-accounts#stocks-note"),
-    TANGIBLE_ASSETS_NOTE("small-full-accounts-note#tangible-assets");
+    TANGIBLE_ASSETS_NOTE("small-full-accounts-note#tangible-assets"),
+    EMPLOYEES_NOTE("small-full-accounts-note#employees-note");
```

Comment:
Do we need 'note' in both parts of the link? We should be consistent with the format of these. For reference, according to the spec the tangible link should be of the format: `small-full-accounts-note#tangible-assets` so I'd expect this to be `small-full-accounts-note#employees`

Reply:
It really should be consistent - looks like stocks and intangible don't have note in the prefix either, but the spec says they should, so I'll update them all to match the spec - "small-full-accounts-note#foo"